### ISSUE TITLE
textedit always one line too short

### ISSUE THREAD

**Issue by hramrach** [Lexikos/AutoHotkey_L#40]
Hello,

I am using AHK 1.1.22.2 to display the output of a script like this:

#SingleInstance force
#i::
RunWait, %comspec% /c C:\windows\sysinfo.rb > C:\temp\%USERNAME%sysinfo.txt,, Hide
FileRead, si, C:\temp\%USERNAME%sysinfo.txt
FileDelete, C:\temp\%USERNAME%sysinfo.txt

Gui, sysinfo: New, +AlwaysOnTOp -MinimizeBox +ToolWindow, sysinfo
Gui, Add, Edit, ReadOnly, %si%
Gui, Add, Button,, Hide

Gui, Show
Return

The dialog shows a text edit which has a scrollbar because the last empty line is not accounted for in calculating the textedit height.

![bez nazvu](https://cloud.githubusercontent.com/assets/787652/8451042/46d498ec-1fe0-11e5-96ff-d7053005e64f.png)

**Comment by joedf** [comment]
change
```AutoHotkey
Gui, Add, Edit, ReadOnly, %si%
```
to
 ```AutoHotkey
Gui, Add, Edit, ReadOnly, % trim(si,"`n")
```

**Comment by Lexikos** [comment]
Fixed by commit 2fe3b50f (v1.1.22.06).
